install/deploy_management_nodes.md

```diff
@@ -571,57 +571,22 @@ The LiveCD needs to authenticate with the cluster to facilitate the rest of the
 
 After the NCNs are booted, the BGP peers will need to be checked and updated if the neighbor IP addresses are incorrect on the switches. Follow the steps below and see [Update BGP Neighbors](../operations/network/metallb_bgp/Update_BGP_Neighbors.md) for more details on the BGP configuration.
 
-1. Make sure the SYSTEM_NAME variable is set to name of your system.
+   1. Make sure the SYSTEM_NAME variable is set to name of your system.
 
-   ```bash
-   pit# export SYSTEM_NAME=eniac
-   ```
-
-1. Determine the IP address of the worker NCNs.
-
-   ```bash
-   pit# grep -B1 "name: ncn-w" /var/www/ephemeral/prep/${SYSTEM_NAME}/networks/NMN.yaml
-   ```
-
-1. Determine the IP addresses for the switches that are peering.
-
-   ```bash
-   pit# grep peer-address /var/www/ephemeral/prep/${SYSTEM_NAME}/metallb.yaml
-   ```
-
-1. Do the following steps for each of the switch IP addresses that you found in the previous step:
-
-    1. Log in to the switch as the `admin` user:
-
-        ```bash
-        pit# ssh admin@<switch_ip_address>
-        ```
-
-    1. Clear the BGP peering sessions by running the following commands. You should see either "arubanetworks" or "Mellanox" in the first output you see when you log in to the switch.
-        - Aruba: `clear bgp *`
-        - Mellanox: First run `enable`, then run `clear ip bgp all`
-
-1. Check the status of the BGP peering sessions by running the following commands **on each switch**:
-    - Aruba: `show bgp ipv4 unicast summary`
-    - Mellanox: `show ip bgp summary`
-
-    You should see a neighbor for each of the workers NCN IP addresses found in an earlier step. If it is an Aruba switch, you will also see a neighbor for the other switch of the pair that are peering.
-
-   At this point the peering sessions with the worker IP addresses should be in `IDLE`, `CONNECT`, or `ACTIVE` state (not `ESTABLISHED`). This is due to the MetalLB speaker pods not being deployed yet.
-
-   You should see that the `MsgRcvd` and `MsgSent` columns for the worker IP addresses are 0.
+      ```bash
+      pit# export SYSTEM_NAME=eniac
+      ```
 
-1. If the neighbor IP addresses do not match the worker NCN IP addresses, use the helper script for Mellanox and CANU (Cray Automated Network Utility) for Aruba.
+   1. Determine the IP address of the worker NCNs.
 
-   1. This command will list the available helper scripts.
       ```bash
-      pit# ls -1 /usr/bin/*mellanox_set_bgp_peer*py
+      pit# grep -B1 "name: ncn-w" /var/www/ephemeral/prep/${SYSTEM_NAME}/networks/NMN.yaml
```

Comment:
Do we need the worker NCN IPs still?

Reply:
I see now after expanding below that we need them for the step of checking the status.